pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -158,6 +150,8 @@ private synchronized void removeAllTasksFromCronExpressions(String tableWithType
       if (jobKey.getName().equals(tableWithType)) {
         try {
           _scheduledExecutorService.deleteJob(jobKey);
+          _controllerMetrics.addMeteredTableValue(getCronJobName(tableWithType, jobKey.getGroup()),
+              ControllerMeter.CRON_SCHEDULER_JOB_SCHEDULED, -1L);
```

Comment:
I feel that `CRON_SCHEDULER_JOB_SCHEDULED` be more of a gauge other than a meter. E.g., meter is measured on per minute basis. Gauge is like a variable storing the current value.

Reply:
make sense, moved it.